src/main/java/org/hl7/davinci/priorauth/ProcessClaimItemTask.java

```diff
@@ -108,7 +124,22 @@ private boolean process() {
 
         Disposition itemDisposition;
         if (!itemIsCancelled) {
-            itemDisposition = PriorAuthRule.computeDisposition(this.bundle, this.item.getSequence());
+            // here is the place to switch
+            if (rulesEngine.equals("internal")) {
+                itemDisposition = PriorAuthRule.computeDisposition(this.bundle, this.item.getSequence());
+            } else {
+                String URL = PropertyProvider.getProperty("url");
+                try {
+                    itemDisposition = sendAndGetDisposition(this.bundle, this.item.getSequence(), URL);
```

Comment:
This config should be more detailed than just "url". I also don't think we need two properties. The "rules_engine" property could be set to "internal" or a URL. That way the if/else remains the same we don't need another config
```suggestion
                try {
                    itemDisposition = sendAndGetDisposition(this.bundle, this.item.getSequence(), rulesEngine);
```